pypeit/core/extract.py

```diff
@@ -1517,8 +1519,12 @@ def objfind(image, thismask, slit_left, slit_righ, inmask=None, fwhm=3.0, use_us
         plt.xlabel('Approximate Spatial Position (pixels)')
         plt.ylabel('F/sigma (significance)')
         plt.title(qa_title + ': Slit# {:d}'.format(specobj_dict['SLITID']))
-        viewer, ch = display.show_image(image*(thismask*inmask))
-        plt.show()
+        if debug_all:
```

Comment:
So if outfile_objprof is not None, but show_peaks is False, then the plot does not get made. I think we need to think through the QA logic here. 

The point is we either want to do some full interactive plotting, or we want to make QA files and we should have the flexibility to do one, the other, or both.